packages/@o3r/test-helpers/src/test-environments/create-test-environment-angular.ts

```diff
@@ -140,7 +140,7 @@ export async function createTestEnvironmentAngular(inputOptions: Partial<CreateT
       const packageJson = JSON.parse(packageJsonString);
       writeFileSync(workspacePackageJsonPath, JSON.stringify({
         ...packageJson,
-        scripts: {...packageJson.scripts, build: getPackageManager() === 'npm' ? 'npm run build --workspaces' : 'yarn workspaces foreach run build'},
+        scripts: {...packageJson.scripts, build: getPackageManager() === 'npm' ? 'npm run build --workspaces' : 'yarn workspaces foreach -A run build'},
```

Comment:
the `-A` is also working with yarn3 + workspaces plugin?

Reply:
Yes it is part of the yarn 3 documentation and this is only used in tests